addons/payment/models/res_company.py

```diff
@@ -50,3 +50,17 @@ def _install_modules(self, module_names):
         modules_sudo = self.env['ir.module.module'].sudo().search([('name', 'in', module_names)])
         STATES = ['installed', 'to install', 'to upgrade']
         modules_sudo.filtered(lambda m: m.state not in STATES).button_immediate_install()
+
+    @api.model_create_multi
+    def create(self, vals_list):
```

Comment:
CRUD methods should be placed before action (`_run_payment_onboarding_step`) methods.